src/lib/userStorage/__tests__/FeedStorage.js

```diff
@@ -0,0 +1,33 @@
+import 'fake-indexeddb/auto'
```

Comment:
please
1.  merge master to your branch and look at user profile storage test. some polyfills (from entries, indexed db was adjusted and moved to the initTest.js)
2. re use initUserStorage as it's done in the user profile storage

Reply:
Changed in 781edbc9e3778f1c7a05675c3423efe7244504a9